src/core/utils.js

```diff
@@ -900,10 +900,63 @@ export function showWarning(message, pluginName, options = {}) {
 }
 
 /**
- * Creates a quick markdown link to a property in the docs.
+ * Makes a string `coded`.
  *
- * @param {string} prop ReSpec configuration property to link to in docs.
+ * @param {string} item
+ * @returns {string}
  */
-export function docLink(prop) {
-  return `[\`${prop}\`](https://respec.org/docs/#${prop})`;
+export function toMDCode(item) {
+  return String(item) ? `\`${item}\`` : "";
```

Comment:
JSDoc + TS will ensure we only pass a string
```suggestion
  return item ? `\`${item}\`` : "";
```